runtimes/runtimes/lsp/router/lspServer.ts

```diff
@@ -89,6 +89,17 @@ export class LspServer {
         token: CancellationToken
     ): Promise<PartialInitializeResult | ResponseError<InitializeError> | undefined> => {
         if (!params.initializationOptions?.aws) {
+            if (this.lspConnection?.telemetry) {
```

Comment:
Why would `lspConnection` connection be undefined given it's expected in the constructor? `lspConnection` is a must - nothing works without it. `telemetry` is also expected to be defined. I do not think we should check everything for `undefined`.